index.js

```diff
@@ -3,18 +3,24 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 1", "color: red") 
 
+var counter = 1;
+var counter = 2;
```

Comment:
`var` works here, but get in the habit of using `let` instead (`var` has some tricky behavior, so you won't see it used as much in modern Javascript). Here's the corrected code:
```js
let counter = 1 // create the variable 
counter = 2      // reassign the variable
```